src/test/java/de/zalando/ep/zalenium/dashboard/LocalDashboardTest.java

```diff
@@ -36,54 +36,76 @@
     @Before
     public void initDashboard() throws IOException {
         ti.setVideoRecorded(true);
-        Dashboard.setExecutedTests(0, 0);
+        LocalDashboard.setExecutedTests(0, 0);
         TestUtils.ensureRequiredInputFilesExist(temporaryFolder);
         CommonProxyUtilities proxyUtilities = TestUtils.mockCommonProxyUtilitiesForDashboardTesting(temporaryFolder);
-        Dashboard.setCommonProxyUtilities(proxyUtilities);
+        LocalDashboard.setCommonProxyUtilities(proxyUtilities);
     }
 
     @After
     public void restoreCommonProxyUtilities() {
-        Dashboard.restoreCommonProxyUtilities();
+        LocalDashboard.restoreCommonProxyUtilities();
     }
 
     @Test
     public void testCountOne() {
-        Dashboard.updateDashboard(ti);
-        Assert.assertEquals(1, Dashboard.getExecutedTests());
-        Assert.assertEquals(1, Dashboard.getExecutedTestsWithVideo());
+        LocalDashboard dashboard = new LocalDashboard();
+        try {
+            dashboard.updateDashboard(ti);
+            Assert.assertEquals(1, LocalDashboard.getExecutedTests());
+            Assert.assertEquals(1, LocalDashboard.getExecutedTestsWithVideo());
+        }
+        catch(Exception e) {
+            Assert.fail(e.toString());
+        }
     }
 
     @Test
     public void testCountTwo() {
-        Dashboard.updateDashboard(ti);
-        Dashboard.updateDashboard(ti);
-        Assert.assertEquals(2, Dashboard.getExecutedTests());
-        Assert.assertEquals(2, Dashboard.getExecutedTestsWithVideo());
+        LocalDashboard dashboard = new LocalDashboard();
+        try {
+            dashboard.updateDashboard(ti);
+            dashboard.updateDashboard(ti);
+            Assert.assertEquals(2, LocalDashboard.getExecutedTests());
+            Assert.assertEquals(2, LocalDashboard.getExecutedTestsWithVideo());
+        } catch (Exception e) {
```

Comment:
Why do you catch the exception and rethrow it? I don't think you need to catch the exception at all. All it means is that the test errors will be harder to find the actual cause because the original exception will be lost.